app/views/npq_separation/admin/applications/show.html.erb

```diff
@@ -197,3 +206,52 @@
     end
   end
 %>
+
+<hr class="govuk-section-break govuk-section-break--l">
+
+<h2 class="govuk-heading-m govuk-!-margin-top-9">Declarations</h2>
+
+<% @declarations.each do |declaration| %>
+  <%= govuk_summary_card(title: declaration.declaration_type.humanize) do |summary_card| %>
+    <%= summary_card.with_summary_list do |sl| %>
+      <% sl.with_row do |row| %>
+        <% row.with_key(text: "Declaration ID" ) %>
+        <% row.with_value(text: declaration.id) %>
+      <% end %>
+
+      <% sl.with_row do |row| %>
+        <% row.with_key(text: "Declaration type") %>
+        <% row.with_value(text: declaration.declaration_type.humanize) %>
+      <% end %>
+
+      <% sl.with_row do |row| %>
+        <% row.with_key(text: "Declaration date") %>
+        <% row.with_value(text: declaration.declaration_date.to_fs(:govuk_short)) %>
+      <% end %>
+
+      <% sl.with_row do |row| %>
+        <% row.with_key(text: "Lead provider") %>
+        <% row.with_value(text: declaration.lead_provider.name) %>
```

Comment:
could we also surface cohort as well for declarations, because a provider can submit a declaration for a cohort, then decide to change that cohort and void the declaration, so would be good to surface the separate cohort on the declaration as the application might be different